zeeguu/core/model/bookmark.py

```diff
@@ -197,6 +204,18 @@ def json_serializable_dict(self, with_context=True, with_title=False):
 
         created_day = "today" if self.time.date() == datetime.now().date() else ""
 
+        # Fetch the BasicSRSchedule instance associated with the current bookmark
+        from zeeguu.core.model import BasicSRSchedule
+        from zeeguu.core.word_scheduling.basicSR.basicSR import MAX_INTERVAL_8_DAY, ONE_DAY
+
+        try:
+            basic_sr_schedule = BasicSRSchedule.query.filter(
+                BasicSRSchedule.bookmark_id == self.id
+            ).one()
+            cooling_interval = basic_sr_schedule.cooling_interval // ONE_DAY
+        except sqlalchemy.exc.NoResultFound:
+            cooling_interval = -1
```

Comment:
I know I was the one to set this to -1, but it might be better to set it to a None so it's clear that it's not set or doesn't exist. I think it better future proofs in case some implementation would have a negative scale.